pkg/quota/evaluator/core/pods.go

```diff
@@ -58,6 +58,31 @@ var podResources = []api.ResourceName{
 	api.ResourcePods,
 }
 
+// podResourcePrefixes are the set of prefixes for resources (Hugepages, and other
+// potential extended reources with specific prefix) managed by quota associated with pods.
+var podResourcePrefixes = []string{
+	api.ResourceHugePagesPrefix,
+	api.ResourceRequestsHugePagesPrefix,
+	api.ResourceLimitsHugePagesPrefix,
```

Comment:
see prior comment re: limits.

Reply:
tracking a list of well known prefixes is fine for now especially since hugepages-* are kubernetes native.  for extended resources, we would want to integrate with a discovery api.